assets/src/util/assignment.js

```diff
@@ -21,15 +21,11 @@ const setAssigmentGoalInputState = (key, assignments, inputFocus) => {
 }
 
 const setAssignmentGoalGrade = (key, assignments, goalGrade) => {
-  // Use decimal place of pointsPossible if it's a decimal; otherwise, round to nearest tenth
-  const placeToRoundTo = (String(assignments[key].pointsPossible).includes('.'))
-    ? getDecimalPlaceOfFloat(assignments[key].pointsPossible) : 1
-
```

Comment:
I think you are right that we need to remove this here and store values at some pre-determined precision and then instead round right before displaying. However, I'm concerned that removing this entirely might re-introduce the bug that I was trying to resolve with this code in the first place . See this comment from your Assignment Planning PR: https://github.com/tl-its-umich-edu/my-learning-analytics/pull/702#issuecomment-580877948

I think we still need some way to sometimes show decimal values with greater precision than a tenth in the UI. Maybe this logic that you removed could also get pushed into `AssignmentTable.js` somewhere, just with like the rounding.

Reply:
You can re-produce the bug I describe in that comment in this issue branch by modifying the points for an assignment to be `25.75`, or something similar.

Reply:
Okay I'll try to take a look at this next week.

Reply:
@ssciolla I think I fixed this now - could you review again?

Reply:
Hi @justin0022, sure, I'll try to get to this today.